### DISCUSSION TITLE
Schedules direct

### DISCUSSION THREAD

**Discussion by Buscochevy** [OpenSageTV/Community#109] [Guide/EPG]
I have lost all of my schedules direct data on my dish network. My local stations are all working in the guide. When I go into setup and select video sources and then my tuner and then select epg lineup and then schedules direct. I get an error, there was a problem communicating with the schedules direct server. Is this anything with sage?
I am not sure what to look for now. I went on SD and I did not see my dish schedule there. Then I seen I needed to activate my membership, ( I don't understand that as my suscription is good until March. I did ad my dish network back in but get the error as menchened above.
Anyone else had a problem like this?
Thanks
Wayne

**Comment by wayner9** [comment]
Try contacting SD.  The guy who runs Schedules Direct, Robert, is very responsive.
